src/change-detecting-filesystem-win32.cpp

```diff
@@ -3,7 +3,7 @@
 
 #if defined(_WIN32)
 
-#include <Windows.h>
+//#include <Windows.h>
```

Comment:
Doesn't this file use Windows APIs? We should `#include` the headers for the functions and classes this module uses.

Reply:
This file includes other files that include the necessary Windows APIs. Is it good form to explicitly include APIs that are used by this file?

Reply:
> Is it good form to explicitly include APIs that are used by this file?

Yes. [Include what you use](https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md).